install.sh

```diff
@@ -2931,7 +2936,7 @@ find_git_hook_templates() {
     printf "Do you want to set up a new Git templates folder? [y/N] "
     read -r SETUP_NEW_FOLDER
 
-    if [ "${SETUP_NEW_FOLDER}" = "y" ] || [ "${SETUP_NEW_FOLDER}" = "Y" ]; then
+    if [ "${SETUP_NEW_FOLDER}" = "y" ] || [ "${SETUP_NEW_FOLDER}" = "Y" ] || is_non_interactive; then
```

Comment:
I'd pull out this
```shell
if is_non_interactive; then
  setup_new_templates_folder
  if [ "$TARGET_TEMPLATE_DIR" != "" ]; then return; fi
fi
```
to before `# 4. try to search for it on disk` so we don't ask questions that won't get answers anyway.